gonzo/clouds/compute.py

```diff
@@ -0,0 +1,254 @@
+import logging
+
+from libcloud.compute.types import Provider as ComputeProvider
+from libcloud.compute.providers import get_driver as get_compute_driver
+
+
+logger = logging.getLogger(__name__)
+backends = {}
+
+
+def backend_for(provider):
+    def wrapper(cls):
+        backends[provider] = cls
+        return cls
+    return wrapper
+
+
+class Cloud(object):
+    compute_session = None
+
+    @classmethod
+    def from_config(cls, cloud_config, region):
+        try:
+            backend = cloud_config['BACKEND']
```

Comment:
see e.g. http://gonzo.readthedocs.org/en/latest/configure.html which includes the file above